.github/workflows/build-jvm.yml

```diff
@@ -14,21 +14,21 @@ jobs:
           java-version: '17'
       - name: Build jar
         run: ./gradlew assemble
-      - name: Upload jar vclib
+      - name: Upload jar ckib
         uses: actions/upload-artifact@v3
         with:
-          name: vclib
+          name: ckib
           path: |
-            vclib/build/libs/*jar
-      - name: Upload jar vclib-aries
+            ckib/build/libs/*jar
+      - name: Upload jar ckib-aries
         uses: actions/upload-artifact@v3
         with:
-          name: vclib-aries
+          name: ckib-aries
```

Comment:
typos everywhere?

Reply:
what in the…? i could swear I used the same search&replace expressions as in the other files